weblate/auth/models.py

```diff
@@ -361,7 +367,7 @@ class User(AbstractBaseUser):
 
     EMAIL_FIELD = "email"
     USERNAME_FIELD = "username"
-    REQUIRED_FIELDS = ["email", "full_name"]
+    REQUIRED_FIELDS = ["email", "full_name", "commit_email"]
```

Comment:
Is it really required? It can be blank according to the model.

Reply:
If there's no fallback, it is.